src/Rule.php

```diff
@@ -73,6 +82,13 @@ public function translateMessage(string $message, array $arguments = []): string
         );
     }
 
+    public function skipOnError(bool $value): self
```

Comment:
If you have a new instance: `withSkipOnError()`, but I suggest that you do not have a new instance here:
```php
public function skipOnError(bool $value): self
{
        $this->skipOnError = $value;
        return $this;
}
```